test/JDBC/jdbc_schedule

```diff
@@ -39,6 +39,9 @@ ignore#!#sys-types-before-dep-vu-cleanup
 ignore#!#sys-table_types-before-dep-vu-prepare
 ignore#!#sys-table_types-before-dep-vu-verify
 ignore#!#sys-table_types-before-dep-vu-cleanup
+ignore#!#GRANT_SCHEMA-vu-prepare
+ignore#!#GRANT_SCHEMA-vu-verify
+ignore#!#GRANT_SCHEMA-vu-cleanup
```

Comment:
For the objects where the permission is granted on the older version, and REVOKE on schema performed on the newer version then permission will be revoked from those older objects. These tests are specifically for Upgrade scenario and it will fail in the current release.